template.jinja2

```diff
@@ -50,6 +52,8 @@ ios-{{ which }}-mode-{{ color }}{{ suffix }}:
   paper-card-background-color: var(--ha-card-background)
   rgb-card-background-color: {{ rgb_card_background }} # see https://github.com/basnijholt/lovelace-ios-themes/issues/60
   ha-card-border-width: 0  # https://github.com/basnijholt/lovelace-ios-dark-mode-theme/issues/82#event-7732695357
+  ha-card-backdrop-filter: blur(20px)
```

Comment:
Note that using a backdrop filter breaks all sorts of dropdowns and popup menus (because the library HA uses it pretty old and does not support it), so it makes themes quite hard to use for editing automations and more advanced stuff. It's great for viewing though.

See https://github.com/Nezz/homeassistant-visionos-theme/issues/2 and https://github.com/Nezz/homeassistant-visionos-theme/issues/4

Reply:
Yes I [found that problem](https://github.com/basnijholt/lovelace-ios-themes/pull/79#issuecomment-2336863475) on your repo and already encountered it myself in testing.

I think for the time being, I cannot make the switch here as *many* people will be unhappy, haha.